ferry.go

```diff
@@ -1158,3 +1161,39 @@ func (f *Ferry) checkSourceForeignKeyConstraints() error {
 
 	return nil
 }
+
+func (f *Ferry) GetTotalRowsAndBytesMap() (totalRowsPerTable map[string]uint64, totalBytesPerTable map[string]uint64) {
+	totalRowsPerTable = make(map[string]uint64)
+	totalBytesPerTable = make(map[string]uint64)
+
+	for _, table := range f.Tables {
+		query := fmt.Sprintf(`
+		SELECT table_rows, data_length
+		FROM information_schema.tables
+		WHERE table_schema='%s'
+		AND table_name='%s'
+		`,
+			table.Schema,
+			table.Name,
+		)
+
+		rows, err := f.SourceDB.Query(query)
+		defer rows.Close()
+
+		var totalRows int
+		var totalBytes int
+
```

Comment:
When there are no rows in the table mysql will return values for `totalRows` and `totalBytes` as `NULL` , and this will create panic in the main process if not handled correctly. 

As a result `totalRows` and `totalBytes` should be `sql.NullInt64`. 

`sql.NullInt64` is a specia type provided by `database/sql` package we use in ghostferry to inteact with mysql. For more information regarding this data type and how NULL values are handled from database rows in go  read this article - https://medium.com/aubergine-solutions/how-i-handled-null-possible-values-from-database-rows-in-golang-521fb0ee267